### DISCUSSION TITLE
Personal versus enterprise for an event

### DISCUSSION THREAD

**Discussion by BethanyWinslow** [mozilla/hubs#3328] [Hubs Cloud Questions] [answered]
I've been making Hubs rooms for some people (I work at a University) and had planned on creating a personal account for Hubs Cloud on AWS, which I can't imagine every outgrowing for the foreseeable future. (Until my university decides to host  better virtual graduation ceremonies, ha ha!) But I have a group that is having big all day Zoom event next month (375-400 people total), and I had planned to give them  a bunch of rooms  for their  student presenter teams to use that would let people drop in on them during the "coffee break".  I've been looking at the documentation and while I love the idea of having multiple servers available in enterprise (just in case), it does look like updating the stack for the largest option (c5.24xlarge) before the event should be totally fine. (?) There is no video or anything streaming, or even any object creation or even taking selfies enabled. People just want to talk to other people. I'd have 40 rooms for the teams  of 4 students, and that's seems pretty good for the remaining guests to drop in and out of different rooms to congratulate individual teams.  The whole networking takes place over one hour. We can tolerate some risk, (the conference is on innovation and they understand end users hardware is at play etc.), but I don't want it to be a total flop.  I could go enterprise  for this event, but that honestly will blow my budget and I'm hoping not to do that for just this one event. So I guess my question  is- do you think my plan for the personal one is ok? Obviously having a backup server is better, but for the number of people at that one hour, do you see any red flags? THANK YOU so much for sharing your knowledge with us newbies!

**Comment by robinkwilson** [comment, marked as answer]
You should be fine with a Personal. Also c5.4xlarge or c5.9xlarge would be ample. 

For enterprise, if you want to have the fallback server just in case, you can use 2 app servers + 0 streaming servers (an extra if something goes wrong -- it's like 1 Personal + 1 Personal). In that case c5.2xlarge or c5.4xlarge would work.

Hope this helps!

**Reply by BethanyWinslow** [reply]
Oh thank you so much for your answer Robin!! I met with a colleague who thought my conclusion made sense, and I asked one of my student workers (computer engineering major) to give me feedback too.  I'm that kind of crazy newbie who will try anything without actually knowing what the hell I'm doing or how all that voodoo is working on the inside. (Honestly, if I wasn't I wouldn't use my phone, microwave, or even drive a car!) So I feel a lot more confident about the event, and I'll be excited to promote Hubs as the the easiest to use VR solution for any of my faculty. THANK YOU for such a great platform, and for your answers Robin!!!
